platforms/src/ZkSync/Providers-config.ts

```diff
@@ -13,13 +13,13 @@ export const PlatformDetails: PlatformSpec = {
 
 export const ProviderConfig: PlatformGroupSpec[] = [
   {
-    platformGroup: "zkSync 1.0",
-    providers: [{ title: "Transacted on zkSync 1.0", name: "ZkSync" }],
+    platformGroup: "zkSync Lite",
+    providers: [{ title: "Transacted on zkSync Lite", name: "ZkSyncLite" }],
```

Comment:
I wonder if we should leave the provider name here as just "ZkSync" so that we are able to compare against the existing hashes (the lookup is done by provider name)?